README.adoc

```diff
@@ -137,3 +141,53 @@ To do that, follow these steps:
 
 After you restart the backend, files in the `data` directory will be made available
 in the *Load demo* dropdown.
+
+== Using the application
```

Comment:
Up to this point, the headings have used headline style capitalization, then switches to sentence capitalization from here to the end. I suggest changing all to sentence style, since that's the preferred style, unless headline style somehow dominates elsewhere in the example, then just stick with headline for consistency.

Reply:
Good catch. Can you share any reference explaining why sentence style capitalization in headings is preferred?

Reply:
It is a matter of consistency. A decision was taken in CCS to use sentence style in documentation. I think the IBM Style Guide prefers it too (for context, we were using the IBM Style Guide for some years before the acquisition was announced).